build-system/compile/compile-wrappers.js

```diff
@@ -40,29 +40,10 @@ exports.extension = function (
   latest,
   isModule,
   loadPriority,
-  intermediateDeps,
   opt_splitMarker
```

Comment:
While we're here, looks like this is unused, too?

Reply:
sure. done.